### ISSUE TITLE
Roadmap and contributing

### ISSUE THREAD

**Issue by rtivital** [mantinedev/mantine#144]
We are always looking for your help and contributions to the Mantine codebase, to learn about current open tasks see [Mantine raodmap](https://trello.com/b/e1AT7w4M/mantine-roadmap) – it's a great starting point. If you need any help with your contribution [contact us on Discord](https://discord.gg/eUZpPbpxb4)

Follow these steps to contribute to Mantine:

- Choose task that you want to work on from Backlog column or [help wanted section on Github](https://github.com/mantinedev/mantine/labels/help%20wanted)
- [Create new issue](https://github.com/mantinedev/mantine/issues/new) on GitHub and tell us that you want to build a specific feature 
- Work on your task according to documentation provided in task card
- If you have any questions during development you can join Mantine Discord server and get help from maintainer and community – https://discord.gg/eUZpPbpxb4
- Once your task is done run `npm test` and if everything is fine submit a Pull Request
- If you cannot finish your task or changed your mind – that's totally fine, just let us know in GitHub issue that you've created in first step. Mantine community is friendly – we won't judge or ask any questions if you decide to cancel your submission
- Wait until your code will be reviewed by maintainer and fix all problems found during code review
- You are awesome! Your contribution was merged!
